NEWS.md

```diff
@@ -0,0 +1,243 @@
+# epiprocess 0.5.0.9999 (dev version)
```

Comment:
Note that this is semver-style, and that 0.x.y we are putting breaking&major changes into x and minor and bugfix into y.